src/domain/storage/StorageBucket/StorageConfigContext.tsx

```
@@ -1,18 +1,20 @@
+import { useContext, createContext, PropsWithChildren } from 'react';
+import useStorageConfig, { type StorageConfigOptions, type StorageConfigProvided } from './useStorageConfig';
 
 const StorageConfigContext = createContext<StorageConfigProvided | undefined>(undefined);
 
-export const StorageConfigContextProvider = ({ children, ...props }: PropsWithChildren<StorageConfigOptions>) => {
-  const storageConfig = useStorageConfig(props);
+export const StorageConfigContextProvider = ({ children, ...rest }: PropsWithChildren<StorageConfigOptions>) => {
+  const storageConfig = useStorageConfig(rest)?.storageConfig;
 
-  return <StorageConfigContext.Provider value={storageConfig}>{children}</StorageConfigContext.Provider>;
+  return <StorageConfigContext.Provider value={storageConfig} children={children} />;
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)